pages/pipelines/audit_log.md

```diff
@@ -100,3 +105,142 @@ USER_UPDATED
 ```
 
 You can also set up [Amazon EventBridge](/docs/integrations/amazon-eventbridge) to stream Audit Log events.
+
+
+## Audit logs for secrets
+
+>📘 Audit logs for secrets do not contain the value or sensitive information about the secret.
+
+
+Audit logs record information of transactions in which secrets are accessed or modified. The following events will be logged:
+* `SECRET_CREATED` This triggers an audit log when a user of an organization initiates the creation of a secret. Secrets can only be created by a user. Below are the fields captured in the audit log for this event.
+
+  ```
+  {
```

Comment:
Is this the sort of JSON formatting we want?

Reply:
I'm not sure that this belongs here given that we dont have this level of detail for any of the other audit events described in this section. I would have thought that the GraphQL part of our docs would cover this sort of info